src/components/Referrals/ReferralHeroSection/ReferralHeroSection.spec.tsx

```diff
@@ -29,8 +30,35 @@ jest.mock('../../../modules/routing/locations', () => ({
 }))
 
 describe('ReferralHeroSection', () => {
-  const profileAddress = '0x123456789abcdef'
-  const inviteUrl = `${INVITE_REFERRER_URL}/${profileAddress}`
+  let profileAddress: string
+  let profileAddressWithClaimedName: string
+  let inviteUrl: string
+  let mockAvatarWithClaimedName: Avatar
+  let mockAvatarWithoutClaimedName: Avatar
+
+  beforeEach(() => {
+    profileAddress = '0x123456789abcdef'
+    profileAddressWithClaimedName = '0x123456789abcdef'
+    inviteUrl = `${INVITE_REFERRER_URL}/${profileAddress}`
+
+    mockAvatarWithClaimedName = {
+      userId: profileAddress,
+      name: 'testuser',
+      hasClaimedName: true,
+      profilePictureUrl: 'https://example.com/avatar.png'
+    } as unknown as Avatar
+
+    mockAvatarWithoutClaimedName = {
+      userId: profileAddressWithClaimedName,
+      name: 'testuser',
+      hasClaimedName: false,
+      profilePictureUrl: 'https://example.com/avatar.png'
+    } as unknown as Avatar
+  })
```

Comment:
As these two mocks only belong to the `when avatar has claimed name` and `when avatar has no claimed name`, would you mind moving it the `beforeEach` of each context?